src/main/java/Parser.java

```diff
@@ -0,0 +1,36 @@
+/**
+ * Class to handle user commands.
+ * @author vanGoghhh
+ */
+
+public class Parser {
+
+    /**
+     * Sorts user commands and handle them according to their types.
+     * @param userCommand input command by the user.
+     * @return the types of Command depending on the user's commands.
+     * @throws InvalidCommandException when a non matching command is entered.
+     */
+    protected Command parseCommand(String userCommand) throws InvalidCommandException {
+        String[] command = userCommand.split(" ");
+        if (command[0].equals("done")) {
```

Comment:
I think you can try using the switch statement which could make the code cleaner, as there would not be as many if else statements?